src/sincosint.jl

```diff
@@ -0,0 +1,258 @@
+const pidiv2 = π/2
```

Comment:
I believe this is unnecessary, i.e just using the constant  `π/2` is sufficient and will be compiled to a constant in the function anyway... can check  comparing `@code_llvm` or  similar